torch/csrc/jit/symbolic_script.cpp

```diff
@@ -20,6 +20,31 @@ const std::vector<std::string> functions = {
                 return grad_self, None
 
             return torch.adaptive_avg_pool2d(self, output_size), backward
+
+        def batch_norm(self,
+                       input : Tensor,
+                       weight : Optional[Tensor],
+                       bias : Optional[Tensor],
+                       running_mean : Optional[Tensor],
+                       running_var : Optional[Tensor],
+                       training : bool,
+                       momentum : float,
+                       eps : float,
+                       cudnn_enabled : bool):
+
+            output, save1, save2, impl_idx = torch._batch_norm_impl_index(
+                input, weight, bias, running_mean, running_var, training,
+                momentum, eps, cudnn_enabled)
+            has_weight = weight._defined()
```

Comment:
These should be `x is not None`. If that doesn't work then we need to file an issue.

Reply:
Oh I had no idea I could do that. Will update.